assets/js/src/core/components/editor-tabs/editor-tabs.tsx

```diff
@@ -30,11 +31,27 @@ export interface IEditorTabsProps {
   showLabelIfActive?: boolean
 }
 
+export interface IconWrapperProps {
+  tabKey: string
+  title: string
+  children: React.ReactNode
+}
+
 export const EditorTabs = ({ defaultActiveKey, showLabelIfActive, items }: IEditorTabsProps): React.JSX.Element => {
   const { styles } = useStyle()
   const { detachWidget } = useDetachTab()
   const { id } = useContext(AssetContext)
+  const [openTabKey, setOpenTabKey] = useState<string | null>(null)
+
+  useEffect(() => {
+    if (openTabKey === null && items.length > 0) {
+      setOpenTabKey(items[0].key)
+    }
+  }, [items])
 
+  const onTabClick = (key: string): void => {
```

Comment:
Does this also work with the keyboard? Otherwise we maybe could use the onChange listener instead?